src/components/settings/CategoriesSetting.tsx

```diff
@@ -19,35 +19,35 @@ import Button from '@mui/material/Button';
 import { t as translate } from 'i18next';
 import { ThreeStateCheckboxInput } from '@/components/atoms/ThreeStateCheckboxInput.tsx';
 import { makeToast } from '@/components/util/Toast.tsx';
-import { IncludeInUpdate } from '@/lib/graphql/generated/graphql.ts';
+import { IncludeOrExclude } from '@/lib/graphql/generated/graphql.ts';
 import { TCategory } from '@/typings.ts';
 import { requestManager } from '@/lib/requests/RequestManager.ts';
 import { CheckboxContainer } from '@/components/settings/globalUpdate/CheckboxContainer.ts';
 
-const booleanToIncludeInStatus = (status: boolean | null | undefined): IncludeInUpdate => {
+const booleanToIncludeInStatus = (status: boolean | null | undefined): IncludeOrExclude => {
```

Comment:
the functions should probably be renamed as well (e.g. `booleanToIncludeOrExcludeStatus`)